src/redux/initialState.js

```diff
@@ -1,5 +1,8 @@
 const initialState = {
   home: '',
+  userSignup: '',
```

Comment:
Having three types of state both handling this features implementation is not good practice in that use of one type of state e.g user to handle all user related objects and embedding the error object into the user state will save you a lot of code and increase app efficiency

Reply:
corrected.

Reply:
So

> > I have run the application and realised that the validation put in place once displayed to the user disappears after sometime, I think this is not a good idea in that if the errors are still displayed to the user they can easily implement that, plus it lacks margin bottom
> 
> Let me Implement it.

Implemented